sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcPartitionDiscoverySuite.scala

```diff
@@ -127,7 +189,7 @@ abstract class OrcPartitionDiscoveryTest extends OrcTest {
             i <- 1 to 10
             pi <- Seq(1, 2)
             ps <- Seq("foo", "bar")
-          } yield Row(i, pi, i.toString, ps))
+          } yield Row(i, i.toString, pi, ps))
```

Comment:
are we testing v1 or v2 here?

Reply:
V2.
For V1 we use `OrcV1PartitionDiscoverySuite`.

Reply:
maybe we should put `V2` in the test suite name as well.

Reply:
This is not quite related to this PR. If we are going to use V2 by default, I think the current test suite name is OK.

Reply:
isn't V1 by default now?

Reply:
For read path, it is V2 by default now.

Reply:
let's make a followup PR to put V2 in the test suite name and do not rely on the default config values.